.github/files/process-coverage.sh

```diff
@@ -14,6 +14,14 @@ echo '::group::Copy coverage into artifacts'
 tar --owner=0 --group=0 --xz -cvvf artifacts/coverage.tar.xz coverage
 echo '::endgroup::'
 
-echo '::group::Send coverage to codecov.io'
-bash <(curl -s https://codecov.io/bash) -s ./coverage || echo 'Codecov failed to upload'
-echo '::endgroup::'
+curl -Os https://uploader.codecov.io/latest/linux/codecov
+chmod +x codecov
+
+for SLUG in $(echo $CHANGED | jq -r 'keys[]'); do
```

Comment:
Cleaner to do it like this
```suggestion
for SLUG in $(jq -r 'keys[]' <<<"$CHANGED"); do
```